tests/seeds/books-seed.sql

```
@@ -0,0 +1,138 @@
+   NULL, NULL, 'identità,libertà,modernità', 1, 1),
+  ('Cent''anni di solitudine',  '9788845292453', 1967, 'Italiano', (SELECT id FROM editori WHERE nome='Feltrinelli'),
+   NULL, NULL, 'colombia,magia,storia,famiglia', 1, 1),
+  ('Delitto e castigo',        '9788845261350', 1866, 'Italiano', (SELECT id FROM editori WHERE nome='Feltrinelli'),
+   NULL, NULL, 'Russia,crimine,redenzione,psicologia', 1, 1),
+  ('Le correzioni',            '9788806163389', 2001, 'Italiano', (SELECT id FROM editori WHERE nome='Einaudi'),
+   NULL, NULL, 'famiglia,america,demenza,contemporaneo', 1, 1);
+
+-- ─── AUTHOR–BOOK LINKS ─────────────────────────────────────────────────────
+-- Aldebaran (all 18 volumes → Leo Aldebaran)
+INSERT IGNORE INTO libri_autori (libro_id, autore_id, ruolo, ordine_credito)
+SELECT l.id, a.id, 'principale', 1
```

SQLFluff (4.1.0)

[error] Reuse of column alias 'id' from line 107.

(AL08)